src/main/java/com/kodilla/ecommercee/mapper/ProductMapper.java

```diff
@@ -4,8 +4,10 @@
 import com.kodilla.ecommercee.domain.Group;
 import com.kodilla.ecommercee.domain.Product;
 import com.kodilla.ecommercee.dto.ProductDto;
+import com.kodilla.ecommercee.exception.GroupNotFoundException;
 import com.kodilla.ecommercee.repository.GroupRepository;
 import com.kodilla.ecommercee.repository.GroupRepository;
+import com.kodilla.ecommercee.repository.ProductRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
```

Comment:
Please cleanup imports:
- double import com.kodilla.ecommercee.domain.Group (both not needed)
- import com.kodilla.ecommercee.repository.GroupRepository;
- import com.kodilla.ecommercee.repository.ProductRepository;

Reply:
Fixed in next commit.